packages/bitcoin/src/p2wsh-p2sh-address-gen.spec.ts

```diff
@@ -0,0 +1,182 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Maybe we can get rid of some of these `eslint-disable-next-line`  /`@typescript-eslint/ban-ts-comment` ? 

It would be nice to start off the new packages with better quality